src/babashka/fs.cljc

```diff
@@ -49,6 +49,10 @@
   ([parent child & more]
    (reduce path (path parent child) more)))
 
+(def ^:private path*
```

Comment:
Why is this needed?

Reply:
So we can use `path` as an argument to functions and still call the original `path` function. Is there another way to accomplish this?

```clojure
(defn foo [arg]
  (println "hi" arg))

(def ^:private foo* foo)

(defn bar [foo]
  (foo* foo))
```